### PR TITLE
HOTFIX Monitoring class improvements

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: src/libraries/mysterium-client/monitoring.js [hunk 1]
```diff
@@ -89,21 +89,15 @@ class Monitoring {
   }
 
   _notifySubscribersStatus (isRunning: boolean) {
-    this._subscribersStatus.forEach((callback: StatusCallback) => {
-      try { callback(isRunning) } catch (e) {}
-    })
+    this._subscribersStatus.forEach((callback: StatusCallback) => callback(isRunning))
```
Comment (donce) [inline src/libraries/mysterium-client/monitoring.js]: `forEach` does not ensure that one failing callback doesn't affect others. Example:
```js
['a', 'b', 'c'].forEach((x) => { console.log(x); if (x == 'b') throw new Error('noo') })
```
Output:
```
a
b
Error: noo
    at forEach (repl:1:70)
    at Array.forEach (<anonymous>)
```

`'c'` element is left uncalled.

**Comment by shroomist** [comment]
these changes seem to be merged already. should we close the PR?

**Comment by ignasbernotas** [comment]
@donce @shroomist @mipo47

**Comment by shroomist** [comment]
@ignasbernotas merge this perhaps, before more conflicts appear

**Pull request merged**